From c0a875c0606c78cc0ffbd5b02aa704628ad44aed Mon Sep 17 00:00:00 2001
From: Bin Yang <yangbin@rock-chips.com>
Date: Fri, 7 Oct 2016 14:54:27 +0800
Subject: [PATCH] arm64: dts: rockchip: enable ldo5 and ldo6 in suspend for
 rk3399 mid

Hall-sensor must keep power on in suspend.

Change-Id: I149e6434f793a6686693f2dabe5959814d134c5e
Signed-off-by: Bin Yang <yangbin@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-mid-818-android.dts | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-mid-818-android.dts b/arch/arm64/boot/dts/rockchip/rk3399-mid-818-android.dts
index 7a9ca12a1344..9860dbfcc182 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-mid-818-android.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-mid-818-android.dts
@@ -743,7 +743,8 @@
 				regulator-max-microvolt = <3000000>;
 				regulator-name = "vcc_3v0";
 				regulator-state-mem {
-					regulator-off-in-suspend;
+					regulator-on-in-suspend;
+					regulator-suspend-microvolt = <3000000>;
 				};
 			};
 
@@ -754,7 +755,8 @@
 				regulator-max-microvolt = <1500000>;
 				regulator-name = "vcc_1v5";
 				regulator-state-mem {
-					regulator-off-in-suspend;
+					regulator-on-in-suspend;
+					regulator-suspend-microvolt = <1500000>;
 				};
 			};
 
-- 
2.35.3

